packages/hop-node/src/watchers/BondWithdrawalWatcher.ts

```diff
@@ -262,6 +262,16 @@ class BondWithdrawalWatcher extends BaseWatcher {
     logger.debug('recipient:', recipient)
     logger.debug('transferNonce:', transferNonce)
     logger.debug('bonderFee:', this.bridge.formatUnits(bonderFee))
+
+    let gasPrice : BigNumber
+    const dbTransfer = await this.db.transfers.getByTransferId(transferId)
+    if (dbTransfer?.transferSentTimestamp) {
+      const item = await this.db.gasPrices.getNearest(this.chainSlug, dbTransfer?.transferSentTimestamp)
```

Comment:
I believe the `chain` needs to be the `destinationChain` since, at the time of this call, we are in the context of the source chain watcher but need the `gasPrice` of the destination chain.

I do not know if `this.db.gasPrices` is aware of the other watcher's `gasPrices` db. If not, then I think `this.db.gasPrices` needs to be able to read from the other DBs.

Reply:
1. you're correct, should be destinationChain
2. this.db.gasPrices is a global db and not watcher specific